api/api.go

```diff
@@ -680,6 +683,13 @@ chmod +x /etc/sv/$APP_NAME/run
 
 # Create a link from /etc/service/$APP_NAME -> /etc/sv/$APP_NAME
 ln -s /etc/sv/$APP_NAME /etc/service/$APP_NAME
+
+# Set the X-Soapbox-App-Version HTTP header
+sed -i .bak \
+  "s/proxy_set_header X-Soapbox-App-Version \"latest\"/proxy_set_header X-Soapbox-App-Version \"$RELEASE\"/" \
```

Comment:
This will send `X-Soapbox-App-Version` to the backend (Go app), but don't we want to send it as part of the response to the client? In that case we'd need `add_header` here and in the ansible nginx conf file.